pages/platform/kubernetes-k8s/using-cluster-autoscaler/guide.en-gb.md

```diff
@@ -0,0 +1,189 @@
+
+
+### Scaling up and down
+
+To scale-up a cluster, the cluster autoscale watches the load on your nodes and detects when your cluster has resource constraints (e.g. it cannot schedule nodes because of lack of resources). When such a situation arise, the cluster autoscaler add nodes to your [node pools](../managing-nodes/) to match the demand. 
+
+The cluster autoscaler also works the other way around, monitoring underutilized nodes and decreasing the number of nodes, helping to reduce your costs.
+
+> [!primary]
+>
+> The scaling up is conceptually easier than the scaling down. When the autoscaler detects resource constraints, it *simply* needs to add yet another node (providing it's under the user defined limit of nodes). But in order to scale down, even if most nodes are underutilized, it needs to move all the pods outside of one of the nodes before deleting it.
+>
+> Some considerations like pods using node selectors, anti-affinity, or restrictive PodDisruptionBudget (PDB).
+> 
+> For more information on this factors disrupting the scaling down of the cluster, please see the [What types of pods can prevent CA from removing a node?](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md#what-types-of-pods-can-prevent-ca-from-removing-a-node) page.
+
+In order to scale up and down the cluster, the autoscaler uses user-defined parameters for things like limits (*what are the upper and lower number of nodes limits in your cluster?*), nodes resource thresholds (*at what load level we add or remove a node?*) or time intervals (*how often we scale the cluster?*). These parameters are described in the [Configuring the cluster autoscaler](../configuring-cluster-autoscaler/) guide. 
+
+## Enabling the autoscaler
+
+### Using Kubernetes API
+
+The easiest way to enable the autoscaler is using the Kubernetes API, for example using `kubectl`.
+
+As explained in the [How nodes and node pools work](../managing-nodes/) guides, in your OVHcloud Managed Kubernetes cluster, nodes are grouped in node pools (group of nodes sharing the same configuration). 
+
+Autoscale is configured in a node pool basis, i.e. you don't enable autoscaling on a full cluster, you enable it for one or or more of your node pools. Let's see how you can do it.
+
+If you activate autoscaler on several node pools, each of which could have different type of instance and min and max limits pon node number.
+
+
+When you create your cluster, it's created with a default node pool, and you can add others in the Public Cloud section of the [OVHcloud Control Panel](https://www.ovh.com/manager/cloud/) or directly [using the Kubernetes API](../managing-nodes/). 
```

Comment:
same comments as on the other file describing these :)

Reply:
rewriting that...